plugins/BananaSlug/BananaSlugWebNodeMeasurer.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+'use strict';
+
+const BananaSlugAbstractNodeMeasurer = require('./BananaSlugAbstractNodeMeasurer');
+
+const DUMMY = {
+  bottom: 0,
+  height: 0,
+  left: 0,
+  right: 0,
+  scrollX: 0,
+  scrollY: 0,
+  top: 0,
+  width: 0,
+};
+
+class BananaSlugWebNodeMeasurer extends BananaSlugAbstractNodeMeasurer {
```

Comment:
It makes sense to me.

I want to encapsulate the measurement logic within platform-specific modules.

For example, `BananaSlugWebNodeMeasurer` implements the method to measure a web node. I'd imagine that we can add `BananaSlugIOSViewMeasurer` to measure a view as well.